app/component/stepBackHeader.tsx

```diff
@@ -8,15 +8,22 @@ import { Divider } from "./divider"
 type StepBackHeaderProps = {
   title: string
   children?: React.ReactNode
-  href: Path
+  href?: Path
+  handleOnClick?: () => void
```

Comment:
```suggestion
  onStepBack?: () => void
```

Reply:
And I am pondering that the handler actually is more general and can cover `href` case. Another thing is that they have hidden priority, `href` only takes effect when no handler is given.

But `href` is indeed convenient in some scenario. Would like to know your thoughts.

Reply:
Um... I think I prefer using handler due to the flexibility. 
How about adding the default value for `href`? ([Set default value to href](https://github.com/pilagod/waallet-extension/pull/166/commits/a3043944bfd55511bad60f776ca0949f086c3f57))

Reply:
If we prefer using handler, it would be better to remove the `href` prop and set default location to index by `navigate` (from location hook) in the default handler.

Reply:
Good idea! 
Changed here:
[remove href from StepBackHeader](https://github.com/pilagod/waallet-extension/pull/166/commits/dbee54dbe07cbd3d6b1a0d476d10fe32cf8750ea)